### DISCUSSION TITLE
Blazorise in a blazor hybrid app

### DISCUSSION THREAD

**Discussion by Akdimi** [Megabit/Blazorise#2277] [Q&A]
Hi guys, i want to create a Blazor hybrid App and i was wondering if i can use Blazorise for the shared UI? I know how to use it one Blazor server and WebAssembly. Now I am trying to import Blazorise into the shared project and it installs but when i try to code it doesnt recognise it. Is there something i am missing? Or is it not conpatible yet?

**Comment by stsrki** [comment]
It might be not working because Blazorise uses .Net 5, while the hybrid app uses .NetStandard 2.0. I don't think it would work.
